### ISSUE TITLE
Still relevant on Firefox?

### ISSUE THREAD

**Issue by mhalano** [erkserkserks/h264ify-firefox#14]
I opened up the page https://youtube.com/html5 and all options are marked, so I think if this extension still is relevant on Firefox.

**Comment by palopezv** [comment]
I believe it is very relevant. Most GPUs support h.264 hardware video decoding, while very few, if any support vp9 hardware decoding.

**Comment by lordcrc** [comment]
Without this, my NUC would be useless for viewing YouTube videos.

**Comment by Osahashi** [comment]
I ask myself the same question a few days ago, but after using VP9 for a time I go back to this addon and deactivate the VP9 support on Firefox.

I'm using a VPN for YouTube and with VP9 live streams are laggy and some audio lags, the same stream without VP9 no lag. And to be honest, I didn't see any benefit in VP9. The videos are the same.

**Comment by weedy** [comment]
I open https://youtube.com/html5 and I still have Media Source Extensions checked WITH the addon installed.

**Comment by Osahashi** [comment]
The addon does not block Media Source Extensions, it only blocks VP9. Don't look at the screenshot on the front page, it's outdated. Blocking Media Source Extensions make no sense, you can't block it.

**Comment by palopezv** [comment]
To be clearer: Media Source Extensions are, probably, not what you think, @weedy. MSE is a protocol to transmit video and audio streams separate from each other so that the client, your browser, joins them and plays synced audio and video. What for? To be able to do adaptive bitrate streaming; as you can obseve in YouTube when you right-click and select "Stats for nerds".

**Comment by weedy** [comment]
Well for me my issue is I still get VP9, so I was wondering if they are sneaking in over MSE.

![youtube](https://i.imgur.com/ft9oLZu.png)

Edit: wait no, now it's H264 but it's running on my CPU. What the F
Ignore me

**Comment by Osahashi** [comment]
Yes, its not VP9 what you see. VP9 is always listed as VP9 on Codecs:

![unbenannt](https://user-images.githubusercontent.com/22057956/32941219-56184b16-cb85-11e7-869c-9780ed457c22.JPG)

**Comment by LinuxOnTheDesktop** [comment]
And what about _now_ please, when we have Firefox _80_ which advertises itself as having GPU-accelerated video? Thanks.

**Comment by mhalano** [comment]
Redoing the @LinuxOnTheDesktop question: And what about __now__? FIrefox is in version 91.0, a long road was walked. Is this still relevant? I checked this website and H.264 played fine: http://demo.nimius.net/video_test/

**Comment by palopezv** [comment]
@mhalano the purpose of this extension is to **force** AVC1 streams instead of VP9 or AV1 in YouTube. Hardware decoding is a plus and in general you cannot expect anything different to AVC1. The world is big and most people do not own a 11th gen Core I9.

**Comment by Krawei** [comment]
is this addon still a thing in current Firefox-versions <v110? Or should I use the forks instead? How can I verify in current Firefox-Builds that the plugin is working correctly? There's no "media" section in the developer debugging section in Firefox.
